themes/classic/_dev/js/theme.js

```diff
@@ -26,6 +26,8 @@ import 'expose-loader?Tether!tether';
 import 'bootstrap/dist/js/bootstrap.min';
 import 'flexibility';
 import 'bootstrap-touchspin';
+import 'select2';
+import 'select2/dist/css/select2.min.css';
```

Comment:
Are you really importing a css file into theme.js? :)

Reply:
Don't worry, these files are compiled with webpack and extract from js configuration to a proper css files ;)